ios/FluentUI/People Picker/PersonaListView.swift

```diff
@@ -156,6 +156,7 @@ open class PersonaListView: UITableView {
                 self.searchResultText = "%d results found from directory".localized.formatted(with: self.personaList.count)
             }
         })
+        searchDirectoryState = .idle
```

Comment:
>searchDirectoryState = .idle [](start = 8, length = 28)

is this fixing the bug you mentioned in comments (thanks for that!)? Could you explain a bit more? If this is the right code path, do we still need line 151 that sets the a different state?

Reply:
Yes, so line 151 changes the state to searching, displays a loading spinner on UI and displays count of searched results. After that to reset its state to idle(bring back the search directory button), this state has been added here